docs/deployments/python-packages.md

```diff
@@ -0,0 +1,29 @@
+# Python packages
```

Comment:
Include setup.py for completeness?

Reply:
Good idea, I'll do that in a separate PR